src/trix/elements/trix_editor_element.coffee

```diff
@@ -146,6 +146,13 @@ Trix.registerElement "trix-editor", do ->
         @parentNode.insertBefore(element, this)
         element
 
+  form:
+    get: ->
+      if @hasAttribute("form")
+        @ownerDocument?.getElementById(@getAttribute("form"))
+      else
+        @inputElement?.form || findClosestElementFromNode(this, matchingSelector: "form")
```

Comment:
```suggestion
        @inputElement?.form
```
I think we should solely defer to the input element like we do for the `name` and `value` properties: https://github.com/basecamp/trix/blob/66d26af0fdec6424096459047c964a3f9c16593b/src/trix/elements/trix_editor_element.coffee#L164-L170

Reply:
@javan does that mean that we don't want to support the `[form]` attribute directly on the `<trix-editor>` element?

Would that mean that we'd need to make a complementary change to Action Text's [rich_text_area_tag](https://github.com/rails/rails/blob/e56a5146cf176ed752008dbbeb2cba7d43d9b333/actiontext/app/helpers/action_text/tag_helper.rb#L21-L35) to support forwarding along the `form: "..."` option to the `<input type="hidden">`?

Reply:
I've opened https://github.com/rails/rails/pull/42069 to add support for the `form:` keyword.